tests/eval_tests/tools/tools_bad_function_names.py

```
@@ -0,0 +1,463 @@
+def function_23(celsius: float) -> float:
+    """"
+    :description: Converts temperature from Celsius to Kelvin.
+    :use_case: Use this tool when the user explicitly asks to convert a temperature FROM Celsius TO Kelvin.
+    Look for keywords like 'Celsius to Kelvin', 'C to K', or 'convert NUM C to K'.
+    :param celsius: The temperature in Celsius.
+    :returns: The temperature converted to Kelvin.
+    """
+    return celsius + 273.15
+
+@client_tool
+def function_23(fahrenheit: float) -> float:
```

Ruff (0.11.9)

Redefinition of unused `function_23` from line 255

(F811)